src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -125,3 +128,184 @@ def upload_bundle_to_fhir_server(bundle: dict, access_token: str, fhir_url: str)
     except Exception:
         logging.exception("Request to post Bundle failed for json: " + str(bundle))
         return
+
+
+def export_from_fhir_server(
+    access_token: str,
+    fhir_url: str,
+    export_scope: str = "",
+    since: str = "",
+    resource_type: str = "",
+    poll_step: float = 30,
+    poll_timeout: float = 300,
+) -> dict:
+    """Initiate a FHIR $export operation, and poll until it completes.
+    If the export operation is in progress at the end of poll_timeout,
```

Comment:
Yeah what happens if this is the case?

Reply:
A TimeoutError will bubble up to the function.  The FHIR server will continue to build the export files, but they will not be processed.  There are a few options for change here...
* We could catch and log the error, return an HTTP error code to the caller, but let the function call succeed.  In this case Azure App Insights would record it as a successful request (I think).
* We could cancel the export job and delete the blob container.  

I opted to keep it simple for this iteration since it was intended to be more of a baby step, but I think taking a closer look at this is a good idea for coming sprints.